app/Http/Transformers/PhotoTransformer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Photo;
+use League\Fractal\TransformerAbstract;
+
+class PhotoTransformer extends TransformerAbstract
+{
+    /**
+     * List of resources to automatically include
+     *
+     * @var array
+     */
+    protected $defaultIncludes = [
+        'event',
+    ];
+
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models\Photo $photo
+     * @return array
+     */
+    public function transform(Photo $photo)
+    {
+        return [
+            'event_id' => $photo->event_id,
+            'signup_id' => $photo->signup_id,
+            'northstar_id' => $photo->northstar_id,
+            'campaign_id' => $photo->signup->campaign_id,
+            'campaign_run_id' => $photo->signup->campaign_run_id,
+            'post' => [
```

Comment:
Following up on my above comment about the transformer – I feel like this `post` object would make a lot more sense as the "top level" item, and then a nested `photo` (etc.) for any fields specific to a particular type of thing.